CMakeLists.txt

```diff
@@ -700,16 +700,21 @@ add_custom_target(
   github-cli-release
   DEPENDS ${HERMES_GITHUB_DIR}/${HERMES_CLI_GITHUB_FILE})
 
+set(HERMES_PKG_ROOT ${HERMES_GITHUB_DIR}/package-root)
+set(HERMES_PKG_FRAMEWORK ${HERMES_PKG_ROOT}/destroot/Library/Frameworks/hermes.framework)
+
 add_custom_target(
   hermes-runtime-darwin-cocoapods-release
   COMMAND
     ${CMAKE_COMMAND} --build . --target install/strip
   COMMAND
-    mkdir -p ${HERMES_GITHUB_DIR}/package-root
+    mkdir -p ${HERMES_PKG_ROOT}
+  COMMAND
+    cp -R -f ${CMAKE_INSTALL_PREFIX} ${HERMES_PKG_ROOT}/destroot
   COMMAND
-    cp -R -f ${CMAKE_INSTALL_PREFIX} ${HERMES_GITHUB_DIR}/package-root/destroot
+    find ${HERMES_PKG_ROOT}/destroot/bin -type f ! -name 'hermesc' -delete
```

Comment:
This feels weird.  Instead of doing this, could you make 713 copy the one file you want?  I don't know what else is in destroot, but making the contents of the package inclusionary rather than exclusionary seems easier to understand.

Reply:
Good point 👍